webserver/templates/user/scraper.js

```diff
@@ -148,7 +148,10 @@ function getLastFMPage(page) {
 
 var version = "1.4";
 var page = 1;
-var numberOfPages = parseInt(document.getElementsByClassName("pages")[0].innerHTML.trim().split(" ")[3]);
+var numberOfPages = 1;
+if (document.querySelector('.pages') != null) {
```

Comment:
@pinkeshbadjatiya Please change this to use either `querySelector` or `getElementsByClassName`, not both, and to use `===` equality. Otherwise it's a good patch and we can merge it.